src/components/App/ModalApp.tsx

```diff
@@ -293,6 +293,14 @@ class ModalApp extends Component<Props> {
   ) => {
     if (isValidEnterpriseFeature) {
       this.props.actions.showLogoCobranding(logoCobrandConfig)
+      document.documentElement.style.setProperty(
+        '--darkLogoSrc',
```

Comment:
We are adding a property to the host app `html`, which I don't think is ideal. Also, the property name is not generic enough IMO. Can we add this property just to the onfido wrapper?

Reply:
Agree with @stefaniacardenas comments above. 
An alternative is to do something similar to how customisation options are applied. See `setUICustomizations` in `src/components/Theme/utils.ts`.

Reply:
Updated